### PR TITLE
Feature ken

### PR DESCRIPTION
**Pull request by crate19970523**
fix av actress is english name can't take name error

### CODE REVIEW

File: scripts/parseFace.py [hunk 1]
```diff
@@ -29,6 +29,8 @@ def getGirlName(imgItem):
     altStr = str(imgItem.get('alt'))
     name = re.findall(r"[^\x00-\x7F]+", altStr)
     nameStr = ''
+    if len(name) == 0:
+        nameStr = altStr[0:altStr.find(",")] + ','
```
Comment (crate19970523) [inline scripts/parseFace.py:33]: 後來發現直接取不分開的英文的話會發生女優英文名稱兩段英文而無法完整取得的問題，因此最終採取這個作法

Comment (superj80820) [reply scripts/parseFace.py:33]: Please don't workaround. That make logic confusing.
Just 'select continuous string' by regular expression.

```
altStr = str(imgItem.get('alt'))
    name = re.findall(r"[^\x00-\x7F]+|^[a-zA-Z]+", altStr)
    nameStr = ''
    # if len(name) == 0:
    #     nameStr = altStr[0:altStr.find(",")] + ','
    for i in name:
        nameStr = nameStr + i + ','
    return nameStr
```

Comment (crate19970523) [reply scripts/parseFace.py:33]: 但是有一個案例，他的html tag內容是這樣
```China Dahl, japanese pornstar / av actress. also known as: Ms. Small```([來源](https://tw.wav.tv/actresses/top-trending/page:20))
名字應該是"China Dahl"，用那個方式的話會變成：
```China,https://images15.wav.tv/115/5ad97703117e4d.jpg```

Comment (superj80820) [reply scripts/parseFace.py:33]: What about add 'the space'

```
altStr = str(imgItem.get('alt'))
name = re.findall(r"[^\x00-\x7F]+|^[a-zA-Z ]+", altStr)
nameStr = ''
# if len(name) == 0:
#     nameStr = altStr[0:altStr.find(",")] + ','
for i in name:
    nameStr = nameStr + i + ','
return nameStr
```

Comment (crate19970523) [reply scripts/parseFace.py:33]: 我沒有想到這個方法，馬上修正

Comment (superj80820) [reply scripts/parseFace.py:33]: I think for a while. Maybe we can also parse '-', '(', ')' about English condition

```
altStr = str(imgItem.get('alt'))
name = re.findall(r"[^\x00-\x7F]+|^[a-zA-Z \-\(\)]+", altStr)
nameStr = ''
# if len(name) == 0:
#     nameStr = altStr[0:altStr.find(",")] + ','
for i in name:
    nameStr = nameStr + i + ','
return nameStr
```

Comment (crate19970523) [reply scripts/parseFace.py:33]: 好的

**Pull request merged**